pinot-core/src/main/java/org/apache/pinot/core/common/DataFetcher.java

```diff
@@ -100,6 +101,19 @@ public void fetchIntValues(String column, int[] inDocIds, int length, int[] outV
     _columnValueReaderMap.get(column).readIntValues(inDocIds, length, outValues);
   }
 
+  /**
+   * Fetch the int values from a JSON column.
```

Comment:
nit: is `JSON column` too specific for the method?

Reply:
This is a carry over from before generalising the interface. Will fix all of them.